### ISSUE TITLE
removeMember does not remove member from group in AD

### ISSUE THREAD

**Issue by stuieordie** [txstate-etc/ldap-async#4]
The addMember method is adding users to a group, but the removeMember isn't removing them. 

I dug into it a little bit and found that 

`await ldap.removeMember(user_dn, group_dn)` - does not remove
`ldap.pullAttribute(group_dn, 'member', user_dn)` - does not remove

but going back up to this
`ldap.modify(group_dn, 'delete', { ['member']: user_dn })` - works

Which is strange because I believe the removeMember uses the pullAtrribute, and the pullAttribute uses the modify. I'm connected to an active directory environment.

**Comment by stuieordie** [comment]
Alright - I think I figured it out. The removeMember method doesn't work when you're trying to remove the last member from a group. The following line in the pullAttribute method incorrectly splits the dn into the set on each character, which results in the supplied value not being found, resulting in oldValues === 0.

`const existingValues = new Set((_a = current[attribute]) !== null && _a !== void 0 ? _a : []);`

by logging the existingValues var, I was able to see that when there is more than 1 member, the output is

```
Set(2) {
  'CN=TestUser2,OU=Users,OU=Staff,OU=Restricted,DC=domain,DC=com',
  'CN=TestUser1,OU=Users,OU=Staff,OU=Restricted,DC=domain,DC=com'
}
```
But when there's only 1 member

```
Set(25) {
  'C',
  'N',
  '=',
  'T',
  'e',
  's',
  'T',
  'P',
  'U',
  's',
  'e',
  'r',
  '2',
  ',',
  'S',
  'f',
  'R',
  'i',
  'c',
  'd',
  'D',
  'p',
  'o',
  'n',
  'm'
}
```
